SKILL.tsv

```diff
@@ -3750,38 +3750,38 @@ SKILL_20150414_003749	Use pluvial of alchemy and create a new item.
+SKILL_20150414_003768	Deploy an upright shield to block hostile enemies and their ranged attacks.
+SKILL_20150414_003769	Collect stones to use as bullets for Stone Shot. Cannot be obtained in towns.
 SKILL_20150414_003770	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Charge crossbow with multiple arrows and shoot towards enemy.
 SKILL_20150414_003771	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Remove obstacles in front and attack nearby enemies.
-SKILL_20150414_003772	Move quickly while using catapault attacks.
-SKILL_20150414_003773	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Install rotating trap that inflicts enemy within radius.
-SKILL_20150414_003774	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Install trap that shoots buckshot. Trap is controlled using percussion machine.
-SKILL_20150414_003775	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Install trap that bounces off enemy when stepped on.
+SKILL_20150414_003772	Attack with your crossbow while moving quickly.
+SKILL_20150414_003773	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Set up a rotating trap that inflicts damage to enemies within a radius.
+SKILL_20150414_003774	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Set up a trap that shoots buckshot. Trap is triggered using a detonator.
+SKILL_20150414_003775	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Set up a trap that rebounds enemies when stepped on.
 SKILL_20150414_003776	{memo X}{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Explode ally magic circle and inflict damage to enemy.
-SKILL_20150414_003777	Hide the installed trap from the enemies.
-SKILL_20150414_003778	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Install trap that shoots arrows when touching the wire.
-SKILL_20150414_003779	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Hang bomb on enemy. Enemy explodes when touching other enemies.
+SKILL_20150414_003777	Hide the prepared trap from enemies.
+SKILL_20150414_003778	{#DD5500}{ol}[Physical] - [Pierce]{/}{/}{nl}Set up a trap that shoots arrows when an enemy touches the wire.
+SKILL_20150414_003779	{#DD5500}{ol}[Physical] - [Strike]{/}{/}{nl}Attach a bomb to an enemy. The enemy explodes when it comes in contact with other enemies.
 SKILL_20150414_003780	{memo X}Companion speed +#{CaptionRatio}#% {nl}Duration #{CaptionTime}# seconds
-SKILL_20150414_003781	Companion threats and prevents enemy from approaching. Enemy may fall under Fear.
+SKILL_20150414_003781	Your companion intimidates the enemy and prevents it from approaching. Enemies may fall under Fear.
 SKILL_20150414_003782	Rush Dog
-SKILL_20150414_003783	Make companion rush forward to attack.
-SKILL_20150414_003784	Heal poisoned ally. Poison with higher level than Detox can't be healed but its duration decrease.
+SKILL_20150414_003783	Order your companion to rush forward and attack the enemy.
+SKILL_20150414_003784	Cure a poisoned ally. A poison status with a higher level than Detox can't be healed, however its duration decreases.
```

Comment:
Cure a poisoned ally. A poison status with a higher level than Detox can't be healed, however its duration decreases.

to

Detoxifies/Cures a poisoned ally. A poison status with a higher level than Detoxify cannot be healed, however its duration decreases.